src/BetweenTheBubbles.tbasic

```diff
@@ -1,4 +1,4 @@
-﻿' [+] New inifinte bubbleverse!
+' [+] New inifinte bubbleverse!
```

Comment:
I am not sure I follow changes in this file - are they intentional, please?

Reply:
Just re-save without bom nor the thinAir unwanted chars at the end of script. That's the unwanted result. I can't do nothing about it.

Reply:
I see, thank you for the explanation 👍